modules/monitoring-configuring-metrics-collection-profiles.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_content-type: CONCEPT
+[id="configuring-metrics-collection-profiles_{context}"]
+= Configuring metrics collection profiles
+ 
+[IMPORTANT]
+====
+[subs="attributes+"]
+Using a metrics collection profile is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete.
+Red Hat does not recommend using them in production. 
+These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview[https://access.redhat.com/support/offerings/techpreview].
+====
+
+By default, Prometheus collects all metrics exposed by the default metrics targets in {product-title} monitoring components. 
+However, you might want Prometheus to collect fewer metrics for a cluster in some scenarios:
+
+* Users of a cluster require only platform alerts and have no need for other metrics.
+* A cluster increases in size, and the increased size of the default metrics data being collected now requires increased CPU and memory resources. 
+
+You can use a metrics collection profile to choose to collect either the default amount of metrics data or a minimal amount of metrics data. 
+When you choose to collect a minimal amount of metrics data, basic monitoring features such as alerting continue to work.
+At the same time, the CPU and memory resources required by Prometheus are lowered.
+
+[id="about-metrics-collection-profiles_{context}"]
+== About metrics collection profiles
+
+You can enable one of two metrics collection profiles:
+
+* *Full*: Prometheus collects all metrics data exposed by the different platform components. This setting is the default.
```

Comment:
from https://github.com/openshift/cluster-monitoring-operator/blob/0157b7d01e288eb26ae6e220295b529740902fd3/pkg/manifests/types.go#L24-L27
since the valid value is full,minimal, we should use the same values here

Reply:
fixed